### ISSUE TITLE
Klayout doesn't support hotkeys in this build

### ISSUE THREAD

**Issue by atorkmabrains** [hdl/conda-eda#289]
Hi @proppy 

I was trying the klayout built in the conda environment. It doesn't support hotkeys like the normal one. We need to find a solution for that.

**Comment by proppy** [comment]
what do you mean by `hotkeys`?

**Comment by atorkmabrains** [comment]
Yes Keyborad shortcuts, those are really important for layout editing.

**Comment by atorkmabrains** [comment]
@proppy Any update on this?

**Comment by proppy** [comment]
@atorkmabrains which platform are you trying on?

**Comment by atorkmabrains** [comment]
@proppy Ubuntu 22.04
